src/basic/terminal-util.c

```diff
@@ -1135,3 +1135,15 @@ int open_terminal_in_namespace(pid_t pid, const char *name, int mode) {
 
         return receive_one_fd(pair[0], 0);
 }
+
+bool colors_enabled(void) {
+        char *env;
+
+        env = secure_getenv("SYSTEMD_COLORS");
+
+        if (!env)
```

Comment:
I agree with @zonque that the new line between the env assignment, and the env check shouldn't be there. While this is documented nowhere I think it is a good idea to acquire a value and verify it close to each other, and that means no separating newlines, to indicate the close relation.